ftva_lab_data/views.py

```diff
@@ -147,15 +147,32 @@ def render_search_results_table(request: HttpRequest) -> HttpResponse:
     search = request.GET.get("search", "")
     search_column = request.GET.get("search_column", "")
     page = request.GET.get("page", 1)
-    display_fields = [field for field, _ in COLUMNS]
+    items_per_page = request.GET.get("items_per_page")
 
+    display_fields = [field for field, _ in COLUMNS]
     # If there's a specific search column, use it;
     # otherwise, search in all display fields.
     search_fields = [search_column] if search_column else display_fields
 
     items = get_search_result_items(search, search_fields)
 
-    paginator = Paginator(items, 10)
+    items_per_page_options = [10, 20, 50, 100]  # hard-coded here for now
```

Comment:
See my comment on tests. It's probably worth moving this 1 line to a trivial `view_utils.get_items_per_page_options()`, so it can be included in tests and you can easily determine the default value from the list (assuming it's always intended to be the first in the list... might be better to refactor that too).